### PR TITLE
Update compile and target SDK version

### LINKED ISSUE #457: Increment the compile and target SDK version
**Is it a**

- [x] Bug Report

- [ ] Feature Request

## Bug Report
The compileSdkVersion and targetSdkVersion which are set to versions 26 and 25, respectively, must be changed to 27 as latest Android APIs must be used to make use of the latest features. Along with it build tools version has to be changed and Travis CI has to be reconfigured.

@mhutti1 @kelson42 Can you please review the issue? I will fix it.

### PR DESCRIPTION
**Pull request by abdulwd**
Fixes #457 
Changes: Update compile and target SDK version of the app.

### CODE REVIEW

**Comment by abdulwd** [comment]
@mhutti1 @kelson42 Please review the PR. :)

File: kiwixlib/build.gradle [hunk 1]
```diff
@@ -19,9 +19,9 @@ android {
 }
 
 dependencies {
-  compile fileTree(dir: 'libs', include: ['*.jar'])
-  androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+  implementation fileTree(dir: 'libs', include: ['*.jar'])
+  androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
```
Comment (srv-twry) [inline kiwixlib/build.gradle]: Was changing these lines a part of this issue ?

Comment (abdulwd) [reply kiwixlib/build.gradle]: When we are upgrading the **compileSdkVersion** and the **targetSdkVersion**, we must also upgrade to use the latest Android plugin version which requires a minimum Gradle version. So after upgrading, we must use the latest dependency configuration. And changing one thing and leaving the other often leads to crashes. So changing these lines becomes a part and parcel of this issue and I have tested the app everything is up and running. There's no harm in changing these lines. The PR can be merged safely. :)

Comment (mhutti1) [reply kiwixlib/build.gradle]: Just a side note kiwixlib is separated from the main project and while this code is likely compiled by a sync. It won't be run unless using kiwix-build.

Comment (brijeshshah13) [reply kiwixlib/build.gradle]: @mhutti1 I do not think these changes were necessary as you stated [here](https://github.com/kiwix/kiwix-android/issues/376#issuecomment-365532141). The old configurations would still do the part to the best of my knowledge as they both are 2 separate modules. It's your call.

**Pull request merged**